deployment/volumes/volume-snapshots/snapshot.yaml

```diff
@@ -2,9 +2,9 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: neo4j-data-snapshot
+    name: uploads-snapshot
     namespace: human-connection
   spec:
     source:
-      name: neo4j-data-claim
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-11T13:35:23Z" title="Tuesday, February 11th 2020, 2:35:23 pm +01:00">Feb 11, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,9 +2,9 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: neo4j-data-snapshot
+    name: uploads-snapshot
     namespace: human-connection
   spec:
     source:
-      name: neo4j-data-claim
```


What happened to the neo4j snapshot? Anyways, as far as I remember this was supposed to be edited by the developer.